kitsune/sumo/static/sumo/js/flagit.js

```diff
@@ -50,25 +62,27 @@ document.addEventListener('DOMContentLoaded', () => {
         }
     }
 
-    const reason = updateUrlParameter('get', 'reason');
-    if (reason) {
+    let reason = updateUrlParameter('get', 'reason');
+
+    if (!reason) {
+        reason = 'spam';
+        updateUrlParameter('set', 'reason', reason);
+        reasonFilter.value = 'spam';
+        fetchAndUpdateContent(new URL(window.location.href));
+    } else {
         reasonFilter.value = reason;
     }
+
     reasonFilter.addEventListener('change', async () => {
         const selectedReason = reasonFilter.value;
-        updateUrlParameter('set', 'reason', selectedReason);
-
-        const url = new URL(window.location.href);
-        const response = await fetchData(url);
 
-        if (response) {
-            const data = await response.text();
-            const parser = new DOMParser();
-            const doc = parser.parseFromString(data, 'text/html');
-            flaggedQueue.innerHTML = doc.querySelector('#flagged-queue').innerHTML;
-            disableUpdateStatusButtons();
-            handleDropdownChange();
+        if (isInitialLoad) {
+            isInitialLoad = false;
+            return;
         }
```

Comment:
When I remove this code related to `isInitialLoad`, this PR works great.

Reply:
> When I remove this code related to `isInitialLoad`, this PR works great.

I'll try to get that money for delivery on all my details for the payment prize money for delivery on all my life and I have been looking for a few more wallets to get the money that is in the future for some time to send it to me and I will be able to send it to my bank account details for the payment of money for delivery on all my details for the crypto currency interest rate payment method cash on delivery of this winnings for selling stock will be able to get the latest product of paid vacation to get that money for delivery on all my account details from the play store and different wallets I've attached my resume to this video is not forgetton to find out more information about the money to pay the balance in my bank account details